composer.json

```diff
@@ -32,7 +32,8 @@
         "jackalope/jackalope-doctrine-dbal": "1.1.*|1.2.*",
         "doctrine/phpcr-odm": "~1.3",
         "doctrine/phpcr-bundle": "~1.3",
-        "hautelook/alice-bundle": "~0.2|~1.2"
+        "hautelook/alice-bundle": "~0.2|~1.2",
+        "brianium/paratest": "~0.12.0|~0.13.2"
```

Comment:
okay i ll remove it from json ... but  if one doesn t  install paratest first the command won t work ( no error throwed , just command line error )